int32 goal_count     # Goal (count until this number)
---
int32 final_count    # Final count result
---
int32 current_number # Feedback (current number)

